Dashboards: Make path to default dashboard configurable (#25595)
grafana/grafana: docs/sources/installation/configuration.md

This will restrict users to set the refresh interval of a dashboard lower than given interval. Per default this is 5 seconds.
The interval string is a possibly signed sequence of decimal numbers, followed by a unit suffix (ms, s, m, h, d), e.g. `30s` or `1m`.

### default_home_dashboard_path

> Path to the default home dashboard. If this value is empty, Grafana will default to use StaticRootPath + "dashboards/home.json"

Reviewer: ```suggestion
Path to the default home dashboard. If this value is empty, then Grafana uses StaticRootPath + "dashboards/home.json"
```
